### ISSUE TITLE
Duplicate ids for elements referenced with ARIA

### ISSUE THREAD

**Issue by zaymasood** [pulibrary/dpul#1264]
### Error in spec/features/accessibility_spec.rb
1) accessibility when visiting bulk actions page complies with WCAG

       duplicate-id-aria: IDs used in ARIA and labels must be unique (critical)
       https://dequeuniversity.com/rules/axe/4.3/duplicate-id-aria?application=axeAPI
       The following 1 node violate this rule:

               Selector: form[action="\/slug-3\/bulk_actions\/add_tags"] > .modal-content > .modal-body > .form-group > input[data-autocomplete-fetched="true"]
               HTML: <input type="text" name="tags" id="tags" value="" class="form-control" data-autocomplete-tag="true" data-autocomplete-url="/slug-3/tags.json" style="display: none;" data-autocomplete-fetched="true">
               Fix any of the following:
               - Document has multiple elements referenced with ARIA with the same id attribute: tags

2) accessibility when visiting catalog page with search exhibit catalog path complies with WCAG
       
       duplicate-id-aria: IDs used in ARIA and labels must be unique (critical)
       https://dequeuniversity.com/rules/axe/4.3/duplicate-id-aria?application=axeAPI
       The following 1 node violate this rule:
           
               Selector: form[action="\/slug-4\/bulk_actions\/add_tags"] > .modal-content > .modal-body > .form-group > input[data-autocomplete-fetched="true"]
               HTML: <input type="text" name="tags" id="tags" value="" class="form-control" data-autocomplete-tag="true" data-autocomplete-url="/slug-4/tags.json" style="display: none;" data-autocomplete-fetched="true">
               Fix any of the following:
               - Document has multiple elements referenced with ARIA with the same id attribute: tags

**Comment by thanyabegum** [comment]
Blocked pending https://github.com/projectblacklight/spotlight/issues/2788
